### ISSUE TITLE
Russian translations.

### ISSUE THREAD

**Issue by FelixdelasPozas** [FelixdelasPozas/TrayWeather#54]
Hi, @adem4ik . For version 1.11.0 the user will be able to choose between "Metric" and "Imperial" instead of just Celsius and Fahrenheit. New translations are needed for the Russian language file. 
```
- Measurement units
- Metric system (ºC)
- Imperial system (ºF)
- mm
- inches
- inch
- hPa
- psi
- miles/hour
- miles/h
- met/sec
```
The last one is "meters/second" abbreviated for the weather dialog and "mm" is millimeter. 
Thanks in advance. 

Regards,
Félix.

**Comment by adem4ik** [comment]
```
Единицы измерения
Метрическая система (ºC)
Имперская система (ºF)
мм
дюймы
дюйм
гПа
psi
миля/час
миля/ч
м/с
```
The is no translation for `psi`.

**Comment by FelixdelasPozas** [comment]
Many thanks! Released the [version 1.11](https://github.com/FelixdelasPozas/TrayWeather/releases/tag/1.11.0) with the translations. 

Regards,
Félix.
